shared_requirements.txt

```
@@ -542,3 +542,7 @@ opentelemetry-sdk<2.0.0,>=1.5.0,!=1.10a0
 #override azure-core-experimental azure-core<2.0.0,>=1.25.0
 #override azure-developer-devcenter isodate<1.0.0,>=0.6.1
 #override azure-developer-devcenter azure-core<2.0.0,>=1.24.0
+#override azure-ai-anomalydetector isodate<1.0.0,>=0.6.1
+#override azure-ai-anomalydetector azure-core<2.0.0,>=1.24.0
+#override azure-ai-anomalydetector typing_extensions>=4.3.0
```

```suggestion
#override azure-ai-anomalydetector typing-extensions>=4.3.0; python_version<'3.8.0'
```
It shall keep consistency with `setup.py`: